core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -4341,6 +4371,15 @@ class KafkaApisTest {
         false
       )).thenReturn(group3Future)
 
+      val group4Future = new CompletableFuture[OffsetFetchResponseData.OffsetFetchResponseGroup]()
```

Comment:
I also feel like that we don't fully repro the issue in this one. We also need to test for both the all topics (null) and specific topics (non null) cases. Do you agree?